### PR TITLE
combines jacoco submodule reports into single report

### PR DESCRIPTION
**Pull request by tvaughan77**

### CODE REVIEW

**Comment by tvaughan77** [comment]
@bsiemon I think this works; I wanna merge this and replace the `apply from` line in runbook-service from the current "upside-jacoco" to "upside-jacoco-multimodule"

File: upside-jacoco-multimodule.gradle [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    onlyIf = {
+        true
+    }
+    doFirst {
+        executionData = files(executionData.findAll {
+            it.exists()
+        })
+    }
+}
+
```
Comment (bsiemon) [inline upside-jacoco-multimodule.gradle]: I think you can hook this up so you can do: gradle jacocoRootReport and have it run the test task first then do the report via task dependencies.

Comment (bsiemon) [inline upside-jacoco-multimodule.gradle]: https://docs.gradle.org/current/userguide/tutorial_using_tasks.html#sec:task_dependencies

Comment (bsiemon) [inline upside-jacoco-multimodule.gradle]: Ah, I see you already did that in-line. super!

**Pull request merged**